keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -535,29 +547,79 @@ Disk-pressure eviction:
   - `reason=Evicted`
   - `message=The node was low on resource: ephemeral-storage. (...)`
 - Container status:
-  - `state=Ternminated`
+  - `state=Terminated`
   - `exitCode=137`
   - `reason=Error`
-- Retriable: Unclear, excessive disk usage suggests a bug or misconfiguration.
-  However, a restart on another node may succeed
+- Retriable: Yes
+
+##### Container memory limit exceeded
 
-##### OOM kill
+Linux:
 
 - Reproduction: We run a job with a pod which attempts to allocate more
   memory than constrained in the container spec by `resources.limits.memory`
-- Comments: handled by kubelet
+- Comments: handled by kubelet in `kubelet/kubelet_pods.go` which merges-in the status update from container runtime
 - Pod status:
   - status: OOMKilled
   - `phase=Failed`
   - `reason=`
   - `message=`
 - Container status:
-  - `state=Ternminated`
+  - `state=Terminated`
   - `exitCode=137`
   - `reason=OOMKilled`
-- Retriable: Unclear, but if occurs when
-  `resources.requests.memory=resources.limits.memory` it strongly suggests
-  a software bug.
+- Retriable: No
```

Comment:
are we doing any validation based on this retriable: no/yes classification?

Reply:
There is no validation based on this. It was added to describe the most natural handling in the scenario. It is redundant with the `DisruptionTarget` / `ResourceLimitExceeded` choice and does not follow directly from the experiment so I’m leaning towards removing it completely.

Reply:
removing them is better indeed.

Reply:
Done